simple-local-avatars.php

```diff
@@ -140,6 +140,9 @@ public function get_avatar( $avatar = '', $id_or_email = '', $size = 96, $defaul
 			$local_avatars[$size] = home_url( $local_avatars[$size] );
 		
 		$author_class = is_author( $user_id ) ? ' current-author' : '' ;
+		if( array_key_exists( 'class', $args ) ){
+			$author_class .= ' ' . $args['class'];

```

Comment:
I'm guessing this is here to mimic what core does with `$args['class']`; however, I'm not sure we want to get into adding it in this particular PR without fully considering the other expected args.

Reply:
A full list of args is available here:

https://codex.wordpress.org/Function_Reference/get_avatar

I can make some modifications to support the other properties in $args (size, height, width, etc) and resubmit, though that will take some pretty significant changes to how the <img tag is being rendered.

Reply:
I'll just try duplicating the code from the native `get_avatar()` function:

https://core.trac.wordpress.org/browser/tags/5.1.1/src/wp-includes/pluggable.php#L2546

Reply:
Hey @heyjones thanks for your work here, this looks great! 

Mimicking what `get_avatar` does makes sense, do you think you will be able to finish this up or would you like some help here to get that added?

Reply:
I'm updating this PR to reflect your 2.1.1 update, going to try to get those additional args implemented.